graphdatascience/query_runner/standalone_session_query_runner.py

```diff
@@ -74,3 +74,6 @@ def last_bookmarks(self) -> Optional[Any]:
 
     def set_server_version(self, _: ServerVersion) -> None:
         super().set_server_version(_)
+
+    def clone(self, host: str, port: int) -> "QueryRunner":
```

Comment:
NIT: if you use `from __future__ import annotations` you can skip the quotes here